tools/clang/test/SemaHLSL/hlsl/workgraph/attribute_diags.hlsl

```diff
@@ -1,4 +1,4 @@
-// RUN: %clang_cc1 -fsyntax-only -verify %s
+// RUN: %dxc -Tlib_6_8 -verify %s
```

Comment:
Do we ignore attached validator versioning when using `-verify` option?  If not, these will cause failures when we test against released validators.

Reply:
Ok, it looks like validator version will only impact `CodeGenOpts`, which won't be used in this case, so I think we're fine.